content/applications/general/auth/google.rst

```diff
@@ -64,24 +79,24 @@ Credentials
 
    .. image:: google/credentials-button.png
       :align: center
-      :alt: Credentials button menu
+      :alt: Credentials button menu.
 
-#. Click on *Create Credentials* and select **OAuth client ID**.
+#. Click on :guilabel:`Create Credentials`, and select :guilabel:`OAuth client ID`.
 
    .. image:: google/client-id.png
       :align: center
-      :alt: Oauth client id selection
+      :alt: OAuth client id selection.
 
-#. Select **Web Application** as the Application type. Now configure the allowed pages on which you
-   will be redirected.
+#. Select :guilabel:`Web Application` as the :guilabel:`Application Type`. Now, configure the
+   allowed pages on which Odoo will be redirected.
 
-   In order to achieve this, in the **Authorized redirect URIs** field, enter your database's domain
-   immediately followed by ``/auth_oauth/signin``. For example:
-   ``https://mydomain.odoo.com/auth_oauth/signin``, then click on *Create*.
+   In order to achieve this, in the :guilabel:`Authorized redirect URIs` field, enter the database's
+   domain immediately followed by ``/auth_oauth/signin``. For example:
+   ``https://mydomain.odoo.com/auth_oauth/signin``, then click on :guilabel:`Create`.
 
-   .. image:: google/create-client-id.png
-      :align: center
-      :alt: Creating oauth client id
+#. Now the *OAuth client* has been created. A screen will appear with the :guilabel:`Client ID` and
+   :guilabel:`Client Secret`. Copy (or write down) the :guilabel:`Client ID`, as it will be
+   necessary for the configuration in Odoo, which will be covered in the following steps.
```

Comment:
```suggestion
#. Now that the *OAuth client* has been created, a screen will appear with the :guilabel:`Client ID`
   and :guilabel:`Client Secret`. Copy the :guilabel:`Client ID` for later, as it will be necessary
   for the configuration in Odoo, which will be covered in the following steps.
```